### DISCUSSION TITLE
How to disable links to other pages?

### DISCUSSION THREAD

**Discussion by magdalenagam** [Lailloken/Exile-UI#379] [Q&A] [answered]
I'm having some difficulty seeing the stats on the items, most of the time I click and it shows me the quick links to poeninja etc. Is there any way to disable that? I just want to see the item info with this mod

**Comment by Lailloken** [comment, marked as answer]
In order to keep hotkey-requirements low, some features are activated by tapping a hotkey and others are activated by long-pressing it.

The item-info panel is initially activated by long-pressing the omni-key, or shift-clicking currency onto items (optional setting, requires additional setup). If the panel is already open, a normal omni-key press on an item will load that item's info.

There's currently no option to disable the context-menu with quick-links or change the omni-key behavior in that respect.
